--- apertium-lex-tools-0.1.0/configure.ac.orig	2013-07-16 19:27:57.000000000 +0200
+++ apertium-lex-tools-0.1.0/configure.ac	2015-12-25 20:31:35.556620583 +0100
@@ -17,12 +17,12 @@
               [  --enable-debug  Enable "-g -Wall" compiler options],
               [CXXFLAGS="-g -Wall";CFLAGS="-g -Wall"])
 
-PKG_CHECK_MODULES([LTTOOLBOX], [lttoolbox-3.2 >= required_lttoolbox_version])
+PKG_CHECK_MODULES([LTTOOLBOX], [lttoolbox >= required_lttoolbox_version])
 
 AC_SUBST(LTTOOLBOX_CFLAGS)
 AC_SUBST(LTTOOLBOX_LIBS)
 
-PKG_CHECK_MODULES([APERTIUM], [apertium-3.2 >= required_apertium_version])
+PKG_CHECK_MODULES([APERTIUM], [apertium >= required_apertium_version])
 
 AC_SUBST(APERTIUM_CFLAGS)
 AC_SUBST(APERTIUM_LIBS)
